apps/web/cypress/tests/billing/billing.spec-ee.ts

```diff
@@ -61,6 +91,21 @@ describe('Billing', function () {
   });
 
   it('should display free trial widget after 20 days', function () {
+    cy.intercept('GET', '**/v1/organizations', (request) => {
+      request.reply((response) => {
+        if (!response.body.data) {
+          return response;
+        }
+
+        response.body['data'] = [
+          {
+            ...response.body.data[0],
+            apiServiceLevel: 'business',
+          },
+        ];
+        return response;
+      });
+    }).as('organizations');
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Since this is repeated a couple times, perhaps we could wrap all the tests that need it in a shared `describe` block with its own `beforeEach`?‏